server/services/zigbee2mqtt/utils/features.js

```diff
@@ -88,6 +88,14 @@ const features = {
     min: 0,
     max: 10000,
   },
+  button: {
+    category: DEVICE_FEATURE_CATEGORIES.BUTTON,
+    type: DEVICE_FEATURE_TYPES.BUTTON.CLICK,
+    read_only: true,
+    has_feedback: false,
+    min: 0,
+    max: 1,
+  },
```

Comment:
Is it a sensor ? According to `read_only = true`

Reply:
Button is like a sensor. It sends state to Gladys